docs/deployment.md

```diff
@@ -18,23 +18,26 @@ The Vehicle Edge consists of several docker containers to setup the mandatory pl
 
 ![./assets/plantuml/vehicle_edge_deployment.svg](./assets/plantuml/vehicle_edge_default_deployment.svg)
 
-This diagram shows the 'default' deployment of Vehicle Edge when the docker-compose file _docker-compose.edge.yml_ is used. This deployment uses examples and placeholders. Therefore, it should be seen as an example how it could look like. Packages are used to logically group the docker-containers regarding their scope and feature set.
+This diagram shows the 'default' deployment of Vehicle Edge when the docker-compose file _docker-compose.edge.yml_ is used. Beside the mandatory packages (blue), this deployment uses examples and placeholders (gray). Therefore, it should be seen as an example how it could look like. Packages are used to logically group the docker-containers regarding their scope and feature set.
```

Comment:
Besides? Not sure :)

Reply:
I think it is beside in this case, because it is used as a preposition (at side of/next to).
Source: https://dictionary.cambridge.org/de/grammatik/britisch-grammatik/beside-or-besides